priv/templates/phx.gen.release/rel/server.sh.eex

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+cd -P -- "$(dirname -- "$0")"
+PHX_SERVER=true ./<%= otp_app %> start
```

Comment:
Similar here:

```suggestion
exec PHX_SERVER=true ./<%= otp_app %> start
```

I don't know if `exec` should come before or after the env var.